packages/server/__tests__/db/seeds/fixtures.js

```diff
@@ -134,10 +134,19 @@ const grants = {
     },
 };
 
+const assignedAgencyGrants = [
+    {
```

Comment:
It might be a good idea to follow the pattern of the other fixtures - both for consistency and so you can reference this entry by key in the test [here](https://github.com/usdigitalresponse/cares-grant-opportunities/pull/49/files#diff-92ec996c0f80423a0ae9a1be8c2c6ba52a3b6b1d5572ad33a0fad113aa698047R68)

Reply:
agree, will do